consensus/src/threshold_simplex/types.rs

```diff
@@ -59,6 +59,11 @@ pub trait Seedable<V: Variant> {
     fn seed(&self) -> Seed<V>;
 }
 
+// TODO: better naming
+pub trait Verifiable<V: Variant> {
```

Comment:
TODO: how to handle UX for partial signatures in same way (multiple signatures to verify for an object and don't want to allow duplicates)

Reply:
Complexity is mapping back these parsed values to the underlying message (without requiring underlying message to be copied)